.github/workflows/testing-and-deployment.yml

```diff
@@ -195,15 +189,24 @@ jobs:
         if: ${{ matrix.vtk-version != 'latest' && matrix.vtk-version != '9.2.2' }}
         run: pip install 'matplotlib<3.6'
 
+      - name: Core Testing (no GL)
+        run: python -m pytest --cov=pyvista -v tests/core tests/examples
+
+      - uses: awalsh128/cache-apt-pkgs-action@v1.1.3
+        with:
+          packages: libgl1-mesa-glx xvfb
+          version: 3.0
+
+      - name: Plotting Testing (uses GL)
+        run: xvfb-run python -m pytest --cov=pyvista --cov-append --cov-report=xml --fail_extra_image_cache -v --ignore=tests/core --ignore=tests/examples
```

Comment:
There's no way that installing GL would break non-plotting modules, right? Just wondering whether ignoring the already checked two test subpackages is completely safe.

Reply:
If this is a concern, one possibility is splitting this into two separate jobs that run concurrently.  That is run a separate job for core only and a separate job for core+plotting.  This would not increase the testing times, just the number of concurrent jobs.

Reply:
I'm not concerned about this. I'm going to advocate that we not add additional CI to test/verify this unless it later proves to be a problem. If installing the GL dependencies breaks the `core` module, I think we have bigger problems in VTK, not PyVista